Document/0x06i-Testing-Code-Quality-and-Build-Settings.md

```diff
@@ -215,7 +251,9 @@ $ carthage update --platform iOS
 
 Next, check the Cartfile.resolved for actual versions used and inspect the given libraries for known vulnerabilities.
 
-> Note, at the time of writing of this chapter, there is no automated support for Carthage based dependency analysis known to the authors.
+> Note, at the time of writing this chapter, there is no automated support for Carthage based dependency analysis known to the authors.
```

Comment:
Awesome update! I was just wondering: is there a way to script a curl or something to every package and check for a security patch ^^? I know it is not what we would like to have, but would it be a good idea?

Reply:
Thanks @commjoen, that is a great suggestion! I am least familiar with the Carthage project manager, but I am curious if there is any work on [OWASP Dependency-Check](https://owasp.org/www-project-dependency-check/) to include Carthage. I will look into it!

Reply:
Unfortunately, I was unable to find out if Carthage was on the road map for Dependency-check. However, I may contribute a Carthage analyzer in the future.

Reply:
I created a ticket for this, so we are not missing it. https://github.com/OWASP/owasp-mstg/issues/1697

Reply:
Awesome! Thank you @sushi2k! 🔥